### PR TITLE
Lightmap atlas example

### PR DESCRIPTION
**Pull request by stevenvergenz**

### CODE REVIEW

File: examples/js/stonehenge/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+		undefined,
+		function(err){
+			console.error(err);
+		}
+	);
+
+	// load texture
+	var txLoader = new THREE.TextureLoader();
+	txLoader.load('lightmap.jpg', function(tex){
+		tex.flipY = false;
```
Comment (brianpeiris) [inline examples/js/stonehenge/index.html]: Why is this necessary? Maybe add a comment explaining it.

File: examples/js/stonehenge/index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+						Math.floor(num/8) * .125
+					);
+				}
+				// remaining 11 blocks are in two rows of 6
+				else {
+					mat.lightMap.offset.set(
+						((num-48) % 6) * .125,
+						.75 + Math.floor((num-48)/6) * .125
+					);
+				}
```
Comment (brianpeiris) [inline examples/js/stonehenge/index.html]: Couldn't these offsets exist in the UV coordinates?

File: examples/js/stonehenge/index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Stonehenge</title>
+	<script src='https://cdnjs.cloudflare.com/ajax/libs/three.js/84/three.js'></script>
+	<script src="https://cdn.rawgit.com/mrdoob/three.js/r84/examples/js/loaders/GLTFLoader.js"></script>
+	<script src='https://sdk.altvr.com/libs/altspace.js/2.4.2/altspace.js'></script>
```
Comment (brianpeiris) [inline examples/js/stonehenge/index.html]: Use a relative url to the dist lib

File: examples/js/stonehenge/index.html (hunk 2 above)
Comment (stevenvergenz) [inline examples/js/stonehenge/index.html]: Normally yes, but this example is specifically for texture *atlases*, where all the objects have the same geometry and material, but the texture offset is different.
